zips/draft-ecc-community-and-coinholder.md

```diff
@@ -0,0 +1,188 @@
+
+This option extends the ``FS_DEFERRED`` lockbox funding stream by a further 1,260,000 blocks (approximately 3 years), ending at the 3rd halving.
+
+### Option 2: Additional funding stream
+
+An additional funding stream will be established, consisting of 12% of the block subsidy paid to a 3-of-5 P2SH multisig with keys held by the same Key-holder Organizations as above. The Coin-holder-controlled Fund is considered to include both funds from the ``FS_COINHOLDER`` funding stream, and funds from the one-time lockbox disbursal described above.
+
+This stream will replace the existing ``FS_DEFERRED`` lockbox funding stream [^zip-1015-funding-streams]. That is, the end height of ``FS_DEFERRED`` will be changed to the activation height of this ZIP. The new ``FS_COINHOLDER`` stream will start at the same height and last until the 3rd halving.
+
+### Option 3: Periodic lockbox disbursement
+
+This would require a mechanism to pay from the Deferred Dev Fund Lockbox to a 3-of-5 P2SH multisig with keys held by the same Key-holder Organizations as above, at periodic intervals. [TBD schedule and period.]
+
+### Requirements on use of the Coin-holder-controlled Fund
+
+The Key-Holder Organizations would be bound by a legal agreement to only use funds held in the Coin-holder-controlled Fund according to the specifications in this ZIP, expressed in suitable legal language. In particular, all requirements on the use of Deferred Dev Fund Lockbox funds in ZIP 1015 [^zip-1015-lockbox] MUST be followed for the funds obtained from the one-time disbursal.
+
+The Financial Privacy Foundation (FPF) would administer the Coin-holder-controlled Fund based on decisions taken by coin-holder voting, following a model decided by the community and specified in another ZIP [TBD].
+
+### Disbursement process
+
+1. Anyone can submit a grant application via a process agreed between the FPF, the Zcash Foundation, and the Electric Coin Company.
+2. Grant applications with total below USD 500,000 (or equivalent in another currency) would be directed to ZCG.
+3. For grant applications above this threshold, a 30-day community review and feedback period would start.
+4. If the grant applications should proceed to a coin-holder vote according to the agreed process, then coin-holders would be asked to vote on it.
+5. If the vote passes, then as payments are scheduled for the grant, then the FPF, the Zcash Foundation, and the Electric Coin Company SHOULD sign the corresponding transactions for disbursement from the Coin-holder-controlled Fund.
+
+For coin holder votes, a minimum of 420,000 ZEC (2% of the total eventual supply) MUST be voted, with a simple majority cast in favor, for a grant proposal to be approved. Upon approval, the grants would be paid to the recipient per the terms of the grant proposal.
+
+Coin holders would not be obligated to fund grants and MAY leave the funds at rest.
+
+No organizations or individuals would be restricted from voting their coins.
```

Comment:
This seems like it belongs in the `Requirements` section. It's not really specification.
```suggestion
No organizations or individuals may be restricted from voting their coins.
```